### PR TITLE
Import fixes

### PR DESCRIPTION
**Pull request by jbarnsley10**

### CODE REVIEW

File: test/unit/app/import/robot/register-import.test.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ describe('register import', () => {
     const { errors } = await importRegister([])
 
     expect(errors).toHaveLength(1)
-    expect(errors[0].errors[0].message).toBe('"owner.address.addressLine1" is required')
-    expect(errors[0].errors[1].message).toBe('"owner.address.postcode" is required')
+    console.log('errors', errors)
```
Comment (whitewaterdesign) [inline test/unit/app/import/robot/register-import.test.js:63]: do you still need this?

Comment (jbarnsley10) [reply test/unit/app/import/robot/register-import.test.js:63]: Haha - no. Thanks for the spot

File: test/unit/app/import/robot/importer.test.js [hunk 2]
```diff
@@ -1,76 +1,166 @@
...
-        firstName: String.fromCharCode(150),
-        lastName: String.fromCharCode(150),
-        address: {
-          addressLine1: String.fromCharCode(150),
-          addressLine2: String.fromCharCode(150),
-          town: String.fromCharCode(150),
-          county: String.fromCharCode(150)
+  describe('replaceUnicodeCharacters', () => {
+    test('replaces chars', async () => {
+      const badRow = {
```
Comment (whitewaterdesign) [inline test/unit/app/import/robot/importer.test.js:40]: never would that thought about this, nice one

**Review by whitewaterdesign — APPROVED** [review]
approving, bar the console.log...but looks good

**Pull request merged**